core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -29,13 +31,16 @@ oppia.directive('collectionSummaryTile', [function() {
       getNodeCount: '&nodeCount',
       getLastUpdatedMsec: '&lastUpdatedMsec',
       getThumbnailIconUrl: '&thumbnailIconUrl',
-      getThumbnailBgColor: '&thumbnailBgColor'
+      getThumbnailBgColor: '&thumbnailBgColor',
+      inCreatorDashboard: '=?inCreatorDashboard'
```

Comment:
This is a bit icky because it interferes with the concept of this component being a reusable directive (which should not really be context-aware). I'm going to look through this PR with an eye towards trying to remove this.